apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-disabled-example.component.html

```diff
@@ -0,0 +1,32 @@
+<fdp-form-group #fg>
+    <fdp-form-field
```

Comment:
the example is not showing proper disabled state. Hand cursor is coming on hovering too.